src/constellation/acme.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import constellation
+from constellation.config import (
+    config_dict,
+    config_integer,
+    config_list,
+    config_string,
+)
+
+
+class AcmeBuddyConfig:
+    def __init__(self, data, path):
+        name = config_string(data, [path, "image", "name"])
```

Comment:
```suggestion
    def __init__(self, data, path: list[str]):
        name = config_string(data, [*path, "image", "name"])
```

using this means `path` can be a list, not just one component. The `*` is a splat and will expand the list.